erpc_python/erpc/transport.py

```diff
@@ -29,6 +29,8 @@
 
 import struct
 import serial
+from rpmsg.sysfs import RpmsgEndpoint
```

Comment:
This import will fail if the rpmsg package is not available. Please move it below the rest of the imports and wrap in a try-except statement. You can set a boolean variable to track whether `RpmsgEndpoint` is available, and use it in `RpmsgTransport`.